scripts/cob-command

```diff
@@ -0,0 +1,328 @@
+#!/bin/bash
+blue='\e[1;34m'
+black='\e[0;30m'
+red='\e[0;31m'
+green='\e[1;32m'
+
+standard_user="robot"
+
+ProgName=$(basename $0)
+
+get_clients_list(){
+  IP=$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')
+  client_list=$(nmap --unprivileged $IP-98 --system-dns | grep report | awk '{print $6}' | sed 's/(//g;s/)//g') 
```

Comment:
this can be extended, but I dont think that make sense use this on a local pc, the cob-command is a tmux server over a network with several users, also it is needed a root user with the specific care-o-bot setup.
But you are right the line is not nice,  waiting for the new network configuration to change this (if it is possible)